feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -0,0 +1,324 @@
+# TE-18.1 gRIBI MPLS in UDP Encapsulation and Decapsulation
+
+Create AFT entries using gRIBI to match on next hop group in a
+network-instance and encapsulate the matching packets in MPLS in UDP.
+
+Create a policy routing configuration using gNMI to decapsulate MPLS
+in UDP packets which are sent to a loopback address and apply to
```

Comment:
Pls change this 

From: "to decapsulate MPLS in UDP packets " 
to: "to decapsulate MPLS in UDP and MPLS in GRE packets " 

(anPF needs to decapsulate both type of packets. One coming from Zakim and other from the VM host